wetterdienst/dwd/forecasts/api.py

```diff
@@ -247,6 +247,11 @@ def read_mosmix(self, date: Union[datetime, DWDForecastDate]) -> DWDMosmixResult
 
             df_metadata = df_metadata.join(station_metadata)
 
+            lowercase_columns = True
```

Comment:
This is not finished yet. The `lowercase_columns` option has to be obtained from the runtime configuration settings, i.e. an object attribute in turn propagated by a command line argument.